packages/js/plugins/http/polywrap.plugin.yaml

```diff
@@ -3,3 +3,6 @@ language: plugin/typescript
 name: Http
 module: ./src/index.ts
 schema: ./src/schema.graphql
+import_redirects:
```

Comment:
We should update this format to the latest, since `import_redirects` got renamed to `import_abis`